MvvmCross/Core/Core/ViewModels/MvxViewModelExtensions.cs

```diff
@@ -32,6 +32,10 @@ public static void CallBundleMethods(this IMvxViewModel viewModel, string method
         public static void CallBundleMethod(this IMvxViewModel viewModel, MethodInfo methodInfo, IMvxBundle bundle)
         {
             var parameters = methodInfo.GetParameters().ToArray();
+
+            if (bundle == null && parameters.Count() > 0)
```

Comment:
This check looks odd to me.  I can't quite grok it.  Why is it needed now?  What's changed?